packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/KycVerification/UserAddress/AddressItem/AddressItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { IconChevronRight, PaletteColors, Text } from '@blockchain-com/constellation'
+import styled from 'styled-components'
+
+import { LocationAddress } from '@core/types'
+
+import HighLighted from '../HighLighted'
+
+const Row = styled.div`
+  box-sizing: border-box;
+  border-top: 1px solid ${PaletteColors['grey-000']};
+  &:last-child {
+    border-bottom: 1px solid ${PaletteColors['grey-000']};
+  }
+`
+const FlexWrapper = styled(Row)`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  cursor: pointer;
+`
+
+const DescriptionText = styled.div`
+  display: flex;
+  max-width: 260px;
+`
+
+const DescriptionRowWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`
+const DescriptionRow = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+const AddressItem = ({ address, onClick, searchText }: Props) => {
+  return (
+    <FlexWrapper onClick={() => onClick(address)}>
```

Comment:
since have we need to pass current address if we move it to parent then we will get error like 
```
cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function.
```

but I can rename onCLick prop to be handleClick it's better 👍